segway/gmtk/params.py

```diff
@@ -270,6 +353,26 @@ def uniform_from_shape(cls, shape: int) -> DPMF:
 
         return DPMF(dpmf_values, keep_shape=True)
 
+    def set_dirichlet_pseudocount(self, pseudocount: int):
+        """
+        Set a pseudocount to include in the header alongside the
+        DirichletConst label.
+        """
+        self.pseudocount = pseudocount
+
+    def get_header_info(self) -> str:
+        """
+        Return string representation of own information, for header in
+        input.master section.
+        If the pseudocount attribute was set, include the DirichletConst label
+        and that value in the header before the weights.
+        """
+        line = [str(len(self))]  # dimension
+        if hasattr(self, "pseudocount"):
```

Comment:
same as before, use a `None` design not hasattr

Reply:
See comment on checking argument sum. Changed this to `if self.pseudocount:`